R/dataframe__frame.R

```diff
@@ -1859,6 +1859,36 @@ DataFrame_write_csv = function(
     invisible()
 }
 
+#' @title  Write to parquet file
+#' @inheritParams LazyFrame_sink_parquet
+#'
+#' @rdname IO_write_parquet
+#'
+#' @examples
+#' # write table 'mtcars' from mem to parquet
+#' dat = pl$DataFrame(mtcars)
+#'
+#' destination = tempfile(fileext = ".parquet")
+#' dat$write_parquet(destination)
+DataFrame_write_parquet = function(
+    path,
+    compression_method = "zstd",
```

Comment:
This argument should be called `compression` instead (it can stay as `compression_method` on the Rust side as other functions in Rust use this name)